assets/src/edit-story/elements/media/editPanMovable.js

```diff
@@ -47,21 +51,62 @@ function EditPanMovable({
   const translateRef = useRef([0, 0]);
   const transformFlip = getTransformFlip(flip);
 
-  const update = () => {
+  const {
+    state: { framesLayer },
+  } = useCanvas();
+
+  const update = useCallback(() => {
     const [tx, ty] = translateRef.current;
     fullMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
     croppedMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
-  };
+  }, [croppedMedia, fullMedia, transformFlip]);
 
   // Refresh moveables to ensure that the selection rect is always correct.
   useEffect(() => {
     moveableRef.current.updateRect();
   });
 
+  useKeyDownEffect(
+    framesLayer,
```

Comment:
From discussion with @dvoytenko , this shouldn't be on the `framesLayer`, need to override some other way

Reply:
Really anywhere between the `editLayer` and the actual image editor container is fine for this. The closer the better, of course.

Reply:
It seems like the current behavior of `mousetrap` is to prevent the propagation of captured key events from child to parent, however I've tried using `EditElement`, `EditLayer` and `EditPageArea` to capture these events with no success, the only thing that seems to be working is strangely `framesLayer`. Any idea as to why that happens?

Reply:
Unfortunately no idea. But, IMHO, we should try to start with the `elements/image/edit` (or `elements/media/edit` I think) and debug it. If you'd like feel free to set a handler, push the pull request, and I can help debugging mousetrap.

Reply:
The reason for this is, that `document.activeElement` is the frame element, not the edit element. Thus the keyboard handler needs to be added somewhere above the frame element to be intercepted before getting to the global key handler and adding it to the edit frame or anywhere around that does nothing, as keyboard events aren't handled by this stack.

I'll try to add keyboard focus to the image or a relevant container for the image rather than the frame element.

Reply:
Managed to fix the focus issues by focusing the image itself (and refocusing it whenever the Moveable is invoked).